### ISSUE TITLE
Add clear filters button

### ISSUE THREAD

**Issue by AetherUnbound** [OrcaCollective/OpenOversight#148]
It would be rad to have a "clear filters" button on the frontend search page, which would just clear the filters in the webpage (not actually submit a new search).

**Comment by Krish-bhardwaj** [comment]
Hi  @AetherUnbound  can you assign me this issue and guide me on how to solve this. I am new to open source contributions allow me to solve this problem if possible .

**Comment by AetherUnbound** [comment]
@Krish-bhardwaj Absolutely! Thank you for offering 🙌🏼 If you have any questions about setting up the dev stack, check out [our README.md](https://github.com/OrcaCollective/OpenOversight#development). I'm happy to email you the sample data we use; we _should_ have some checked into the repo but we haven't done that yet (made #151 to track that).

Here are a few other similar PRs that added functionality to the filter pane: #90, #91, #93.

I think this will primarily involve the frontend, as we only need a button in the filter pain that will clear all the current selections. This will probably mean some javascript to remove the current form values - since this is the "list officers" page, whatever JS that's needed should probably go here: https://github.com/OrcaCollective/OpenOversight/blob/main/OpenOversight/app/static/js/list_officer.js

Similarly, this template should have all of the form fields that need to be manipulated: https://github.com/OrcaCollective/OpenOversight/blob/main/OpenOversight/app/templates/list_officer.html

I think that's all that should be involved - if you would like any more context or if there's anything I can elaborate on more, please let me know!

**Comment by gabrielcannes** [comment]
Hi  @AetherUnbound I tried to run this project locally to take a shot at this issue (first time trying to contribute to another repository) but I could not get the list_officer.html to run locally with the CSS. 
Another problem is that for some reason my function OnClick in the newly added clear button its not been triggered (the syntax is correct because it worked on the developer tools inside the browser). Is there any configuration that I need to do before running this project locally?

**Comment by AetherUnbound** [comment]
@gabrielcannes I'm sorry to hear that! I think some of this might be related #151, were you loading any initial data when running locally? I will work on getting a fix for that in today so that mock data is available for testing!

**Comment by AetherUnbound** [comment]
@gabrielcannes I've just added some initial data as part of #152, do you mind checking out that branch and seeing if it works? Instructions for setting up the development environment can be found here: https://github.com/OrcaCollective/OpenOversight#development (if you're on the #152 branch, skip step 2)

**Comment by gabrielcannes** [comment]
@AetherUnbound thank you, I'll look into it.

**Comment by AetherUnbound** [comment]
Awesome, let me know if you run into any more trouble!

**Comment by AetherUnbound** [comment]
Dang! Couple questions for you:
1. Are you on Windows by chance? If so, you may need to try running this via [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install)
2. I've just merged #152, do you mind running `git checkout main && git pull` and trying once more? You shouldn't need to move any files under the new setup!

Let me know if that doesn't work!
